app/views/unattended/provisioning_templates/snippet/create_host_curl.erb

```diff
@@ -0,0 +1,23 @@
+<%#
+kind: snippet
+name: create_host_curl
+model: ProvisioningTemplate
+snippet: true
+-%>
+<%#
+  Creates host via CURL request.
+  Parameters:
+  - :headers [Array]
+  - :organization
+  - :location
+  - :hostgroup
+-%>
+HOSTNAME=$(hostname -f)
```

Comment:
It's a good practice to use long options in scripts, so `--fqdn` instead of `-f`. The same applies to curl.

Reply:
Otoh isn't `--fqdn` just a gnu thing?

Reply:
https://man.openbsd.org/hostname suggests `-f` and `--fqdn` are both non-POSIX options.